src/Microsoft.ML.Data/DataLoadSave/Text/TextLoader.cs

```diff
@@ -474,6 +474,12 @@ public class Options
             [Argument(ArgumentType.AtMostOnce, Name = nameof(Separator), Visibility = ArgumentAttribute.VisibilityType.EntryPointsOnly, HelpText = "Source column separator.", ShortName = "sep")]
             public char[] Separators = new[] { Defaults.Separator };
 
+            /// <summary>
+            /// The character that should be used as the decimal marker.
+            /// </summary>
+            [Argument(ArgumentType.AtMostOnce, Name = "Decimal Marker", HelpText = "Character symbol used to separate the integer part from the fractional part of a number written in decimal form.", ShortName = "decimal")]
+            public char DecimalMarker = Defaults.DecimalMarker;
```

Comment:
Perhaps it's better to only allow the user to specify either "." or "," ??
The Separator (not Separators) option has a documentation which tells the user that only some separators are acceptable, and then in TextLoader.NormalizeSeparator exceptions are thrown if a separator isn't valid. The same could be done with this option.

Reply:
Yup, I thought of this too, but couldn't find a way to enforce '.' and ',' being the only possible decimal markers at this line, so I'll be enforcing this in the codebase below.

---
In reply to: [427820218](https://github.com/dotnet/machinelearning/pull/5145#discussion_r427820218) [](ancestors = 427820218)